index.html

```diff
@@ -71,6 +71,250 @@ <h1 class="headline__title color-dark">
+							</ul>
+							<div class="click">
+								<a href="#" class="button__primary">See project</a>
+							</div>
+						</div>
+					</article>
+					<article class="my_portfolio grid">
+						<header class="nav-port">
+							<img
+								src="img/work/port3.png"
+								alt="portfolio 3"
+								class="port-photo"
+							/>
+						</header>
+							<div class="port-body">
+							<h3 class="port-name color-dark">Tonic</h3>
+							<div class="port-description flex-row align-center">
+								<h4 class="port-invent color-info">Canopy</h4>
+								<ul class="job_done flex-row">
+									<li class="items">Back End Dev</li>
+									<li class="items">2015</li>
+								</ul>
+							</div>
+							<p class="description-main">
+								A daily selection of privately personalized reads; no accounts
+								or sign-ups required.
+							</p>
+							<p class="description-main">
+								A daily selection of privately personalized reads; no accounts
+								or sign-ups required.
+							</p>
+							<ul class="subjects flex-row align-center">
+								<li class="subject">HTML</li>
+								<li class="subject">CSS</li>
+								<li class="subject">JavaScript</li>
+							</ul>
+							<div class="click">
+								<a href="#" class="button__primary">See project</a>
+							</div>
+						</div>
+					</article>
+					<article class="my_portfolio grid">
+						<header class="nav-port">
+							<img
+								src="img/work/port4.png"
+								alt="portfolio 4"
+								class="port-photo"
+							/>
+						</header>
+						<div class="port-body">
+							<h3 class="port-name color-dark">Multi-Post Stories</h3>
+							<div class="port-description flex-row align-center">
+								<h4 class="port-invent color-info">Canopy</h4>
+								<ul class="job_done flex-row">
+									<li class="items">Back End Dev</li>
+									<li class="items">2015</li>
+								</ul>
+							</div>
+							<p class="description-main">
+								A daily selection of privately personalized reads; no accounts
+								or sign-ups required.
+							</p>
+							<ul class="subjects flex-row align-center">
+								<li class="subject">HTML</li>
+								<li class="subject">CSS</li>
+								<li class="subject">JavaScript</li>
+							</ul>
+							<div class="click">
+								<a href="#" class="button__primary">See project</a>
+							</div>
+						</div>
+					</article>
+				</div>
+			</section>
+			<section
+				class="section section__white section__white--top"
+				id="about"
+			>
+				<div class="about flex-column justify-center">
+					<h2 class="about__title color-dark">About Myself</h2>
+					<p class="color-info">
+						Hello I'm a software developer! I can help you build a product,
+						feature or website. Look through some of my work and experience!
+						If you like what you see and have a project you need coded,
+						don't hesitate to contact me.
+					</p>
+					<a href="#">Let's Connect</a>
+					<ul class="social__icons flex-row align-center">
```

Comment:
Please make sure you used the correct font-color from the FGima design
Your design
![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/101523582/2e955db2-5ad5-4906-bfc0-2a38899a5bb5)
Figma design
![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/101523582/06a6f85b-d12d-464c-aa58-c656344efeee)